Modules/CTP/src/RawDataReaderCheck.cxx

```diff
@@ -0,0 +1,147 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   RawDataReaderCheck.cxx
+/// \author My Name
+///
+
+#include "CTP/RawDataReaderCheck.h"
+#include "QualityControl/MonitorObject.h"
+#include "QualityControl/Quality.h"
+#include "QualityControl/QcInfoLogger.h"
+#include "DataFormatsCTP/Configuration.h"
+#include "DataFormatsCTP/RunManager.h"
+// ROOT
+#include <TH1.h>
+#include <TLatex.h>
+#include "DataFormatsCTP/Configuration.h"
+#include "CCDB/BasicCCDBManager.h"
+#include "DataFormatsParameters/GRPLHCIFData.h"
+
+#include <DataFormatsQualityControl/FlagReasons.h>
+
+using namespace std;
+using namespace o2::quality_control;
+using namespace o2::ctp;
+
+namespace o2::quality_control_modules::ctp
+{
+
+void RawDataReaderCheck::configure() {}
+
+Quality RawDataReaderCheck::check(std::map<std::string, std::shared_ptr<MonitorObject>>* moMap)
+{
+  Quality result = Quality::Null;
+
+  // you can get details about the activity via the object mActivity:
+  ILOG(Debug, Devel) << "Run " << getActivity()->mId << ", type: " << getActivity()->mType << ", beam: " << getActivity()->mBeamType << ENDM;
+  // and you can get your custom parameters:
+  ILOG(Debug, Devel) << "custom param physics.pp.myOwnKey1 : " << mCustomParameters.atOrDefaultValue("myOwnKey1", "physics", "pp", "default_value") << ENDM;
+
+  int runNumber = getActivity()->mId;
+
+  o2::ccdb::CcdbApi api;
+  api.init("http://alice-ccdb.cern.ch");
```

Comment:
Do you plan to use this at P2? `alice-ccdb` is not reachable from there.

The recommended way to get objects from CCDB is to use `UserCodeInterface::retrieveConditionAny`, this will use the correct CCDB url. 

An example here:
https://github.com/AliceO2Group/QualityControl/blob/98ce9d57c3decfa11f0119653639e9598825cdc7/Modules/MFT/src/QcMFTDigitCheck.cxx#L123